pkg/policy/l4.go

```diff
@@ -1657,6 +1666,17 @@ func (l4Policy *L4Policy) AccumulateMapChanges(l4 *L4Filter, cs CachedSelector,
 		}
 		epPolicy.policyMapChanges.AccumulateMapChanges(cs, adds, deletes, keysToAdd, value)
 	}
+	l4Policy.mutex.RUnlock()
```

Comment:
nit: Quite a bit of space between the RLock and RUnlock, plus no logic after the unlocking, good candidate to use `defer` to avoid future modification to this function from leaking the lock